plugins/mcreator-localization/lang/texts.properties

```diff
@@ -473,6 +473,7 @@ blockly.block.text_substring.from=from position:
 blockly.block.text_substring.to=to position:
 blockly.block.text_format_number.format=Format number:
 blockly.block.text_format_number.as=as:
+blockly.block.entity_maketamed=Make %1 tamed by %2

```

Comment:
Maybe a suggestion to make the procedure block sound more friendly: `Tame %1 by %2`

Reply:
It's done, I renamed the procedure by `Tame %1 by %2` as suggested